src/styles/blocks/creators.scss

```diff
@@ -0,0 +1,71 @@
+.creators {
+  &__main {
+    &--title {
+      @extend %text-h2;
+      text-transform: uppercase;
+      margin-bottom: 10px;
+  
+      @include Tablet {
+        margin-bottom: 20px;
+      }
+    }
+
+    &--paragraph {
+      font-size: 14px;
+      margin-bottom: 20px;
+
+      @include Tablet {
+        font-size: 16px;
+        margin-bottom: 15px;
+      }
+
+      @include Desktop {
+        margin-bottom: 30px;
+      }
+    }
+
+    &--second-paragraph {
+      margin-bottom: 30px;
+
+      @include Tablet {
+        margin-bottom: 20px;
+      }
+
+      @include Desktop {
+        margin-bottom: 30px;
+      }
+    }
+    
+    &--desktop-paragraph {
+      margin-bottom: 30px;
+    }
+
+    &--link {
```

Comment:
Use a more descriptive name for the link element like '__link' instead of '__main--link'.